src/parser.rs

```diff
@@ -135,11 +137,13 @@ impl Condition {
                 format!("invalid ignore/only filter ending in 'bit': {c:?} is not a valid bitwdith")
             })?;
             Ok(Condition::Bitwidth(bits))
-        } else if let Some(target) = c.strip_prefix("target-") {
-            Ok(Condition::Target(target.to_owned()))
+        } else if let Some(triple_substr) = c.strip_prefix("target-") {
+            Ok(Condition::Target(triple_substr.to_owned()))
+        } else if let Some(triple_substr) = c.strip_prefix("host-") {
+            Ok(Condition::Host(triple_substr.to_owned()))
         } else {
             Err(format!(
-                "invalid condition `{c:?}`, expected `on-host`, /[0-9]+bit/ or /target-.*/"
+                "invalid condition `{c:?}`, expected `on-host`, /[0-9]+bit/, /target-.*/, or /target-.*/"
```

Comment:
Now it says `/target-.*/` twice...

Reply:
Good job at copy paste oli... will fix tomorrow